bcgov/cas-registration
# 195/review facility information

**Card:** https://github.com/bcgov/cas-reporting/issues/195
**Added:** Review Facility Information Page

**Summary of Changes:**

- Created the `ReviewFacilityData` component.
- Added new API routes:
  - Fetch facility data for a specific `facility_id` and report version.
  - Fetch all activities related to a specific `facility_id`.
  - Save facility data.
- Updated `activity_service` to include `get_all_activities` method.
- Updated `report_service`.
- Created `report_facility` schema.
- Added mock fixtures:
  - `report`
  - `report_facility`
  - `report_version`
- Added APIs for:
  - Fetching all activities.
  - Getting report facility data by `facility_id` and report version.
  - Saving report facility data.

**Local Testing Instructions:**

1. Navigate to /bc_obps and run:
    `make reset_db`
    `make run`
    to apply changes to the database.

2. Run the frontend using:
    `yarn dev-all`

3. Insert data into the report_facility_activities table to see selected activities:
    `INSERT INTO erc.report_facility_activities (reportfacility_id, reportingactivity_id)
      VALUES (1, 10), (1, 7), (1, 31);`
      Alternatively, you can add data by selecting activities in the UI.

4. Log in with BCeID to bc-cas-dev and navigate to:
     `/reporting/1/facilities/1/review`
